projects/a51-finance-v3/index.js

```diff
@@ -1,70 +1,154 @@
+    {
+      target: "0x3e0AA2e17FE3E5e319f388C794FdBC3c64Ef9da6",
+      helper: "0xA1d8180F4482359CEb7Eb7437FCf4a2616830F81",
+      startBlock: 12765695,
+    },
+    {
+      target: "0xDFb179526ae303Eea49AC99DD360159C39105828",
+      helper: "0x6e7e838E20ED6657Aaf1166f9B7a845565956F51",
+      startBlock: 13890566,
+    },
+  ],
+  optimism: [
+    {
+      target: "0x525c80e91efe9222de3eae86af69a480fbced416",
+      helper: "0x965356eb2c208ce4130e267342ca720042cce7b2",
+      startBlock: 118360616,
+    },
+  ],
+  polygon: [
+    {
+      target: "0xD4798F142FDb87738eF4eBE82Bd56Eccde19A88C",
+      helper: "0x9c225a02426e3229C073A6132E083561e95000b5",
+      startBlock: 55506149,
+    },
+  ],
+  bsc: [
+    {
+      target: "0x6F2b186e9392042B1edE2D1D1706a3DC4a4725d8",
+      helper: "0x9c225a02426e3229C073A6132E083561e95000b5",
+      startBlock: 37623104,
+    },
+  ],
+  scroll: [
+    {
+      target: "0xA8Dc31c8C9F93dB2e42A5472F580689794639576",
+      helper: "0x965356eb2C208Ce4130E267342cA720042Cce7b2",
+      startBlock: 4846051,
+    },
   ],
-  optimism: [{ target: "0x525c80e91efe9222de3eae86af69a480fbced416", helper: "0x965356eb2c208ce4130e267342ca720042cce7b2", startBlock: 118360616 },],
-  polygon: [{ target: "0xD4798F142FDb87738eF4eBE82Bd56Eccde19A88C", helper: "0x9c225a02426e3229C073A6132E083561e95000b5", startBlock: 55506149 },],
-  bsc: [{ target: "0x6F2b186e9392042B1edE2D1D1706a3DC4a4725d8", helper: "0x9c225a02426e3229C073A6132E083561e95000b5", startBlock: 37623104 },],
-  scroll: [{ target: "0xA8Dc31c8C9F93dB2e42A5472F580689794639576", helper: "0x965356eb2C208Ce4130E267342cA720042Cce7b2", startBlock: 4846051 },],
-  manta: [{ target: "0x69317029384c3305fC04670c68a2b434e2D8C44C", helper: "0xa1d8180f4482359ceb7eb7437fcf4a2616830f81", startBlock: 1834975 },],
-}
+  manta: [
+    {
+      target: "0x69317029384c3305fC04670c68a2b434e2D8C44C",
+      helper: "0xa1d8180f4482359ceb7eb7437fcf4a2616830f81",
+      startBlock: 1834975,
+    },
+  ],
+};
 
 module.exports = {
   doublecounted: true,
-}
+};
 
-Object.keys(config).forEach(chain => {
-  const configs = config[chain]
+Object.keys(config).forEach((chain) => {
+  const configs = config[chain];
   module.exports[chain] = {
     tvl: async (api) => {
-      await Promise.all(configs.map(addConfigTvl))
-
-      async function addConfigTvl({ target, helper, startBlock, topic = DEFAULT_STRATEGY_CREATION_TOPIC }) {
-        const strategyLogs = await getLogs({ target, topic, fromBlock: startBlock, api, })
-        const strategies = strategyLogs.map(log => log.topics[1])
-
-        const pools = []
-        const reservesCalls = []
-
-        const strategyDetails = await api.multiCall({ abi: getStrategiesDetails, target, calls: strategies, })
-
-        strategyDetails.forEach(({ key: { pool, tickLower, tickUpper }, account }) => {
-          pools.push(pool)
+      for (const config of configs) {
+        await addConfigTvl(config);
```

Comment:
I think `await Promise.all()` does the same thing?

Reply:
Yes, but with your change previously the interface wasn't showing the correct tvl for base do you know the reason for that?

Reply:
I think there is a bug/cache issue in the UI. Give it 1-2 days, it usually fixes itself

Reply:
Alright, thanks I'm closing this pr then